<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>David Sugar</author>
 <comment>** Copyright (C) 2011 David Sugar, Tycho Softworks
**
** This file is part of coastal-qt.
**
** Coastal-qt is free software: you can redistribute it and/or modify
** it under the terms of the GNU General Public License as published
** by the Free Software Foundation, either version 3 of the License, or
** (at your option) any later version.
**
** coastal-qt is distributed in the hope that it will be useful,
** but WITHOUT ANY WARRANTY; without even the implied warranty of
** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
** GNU General Public License for more details.
**
** You should have received a copy of the GNU General Public License
** along with coastal-qt.  If not, see &lt;http://www.gnu.org/licenses/&gt;.</comment>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selected</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Other</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="searchBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="searchButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Search</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabs">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="indexTab">
         <attribute name="title">
          <string>Index</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QTableWidget" name="indexTable">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="dragDropOverwriteMode">
             <bool>false</bool>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Section</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionReload"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSections">
    <property name="title">
     <string>Sections</string>
    </property>
    <addaction name="actionAll"/>
    <addaction name="actionOnly"/>
    <addaction name="actionBut"/>
    <addaction name="separator"/>
    <addaction name="actionSection1"/>
    <addaction name="actionSection2"/>
    <addaction name="actionSection3"/>
    <addaction name="actionSection4"/>
    <addaction name="actionSection5"/>
    <addaction name="actionSection6"/>
    <addaction name="actionSection7"/>
    <addaction name="actionSection8"/>
    <addaction name="actionSectionl"/>
    <addaction name="actionSectionn"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionIndex"/>
    <addaction name="actionKeywords"/>
    <addaction name="separator"/>
    <addaction name="actionPaths"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSections"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAll">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;All Sections</string>
   </property>
  </action>
  <action name="actionBut">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>All &amp;But Selected</string>
   </property>
  </action>
  <action name="actionOnly">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Selected &amp;Only</string>
   </property>
  </action>
  <action name="actionSection1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;1: User Commands</string>
   </property>
  </action>
  <action name="actionSection2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;2: System Calls</string>
   </property>
  </action>
  <action name="actionSection3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;3: Subroutines</string>
   </property>
  </action>
  <action name="actionSection4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;4: Devices</string>
   </property>
  </action>
  <action name="actionSection5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;5: File Formats</string>
   </property>
  </action>
  <action name="actionSection6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;6: Games</string>
   </property>
  </action>
  <action name="actionSection7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;7: Miscellaneous</string>
   </property>
  </action>
  <action name="actionSection8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;8: Administration</string>
   </property>
  </action>
  <action name="actionSectionl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;l: Local</string>
   </property>
  </action>
  <action name="actionSectionn">
   <property name="text">
    <string>&amp;n: New</string>
   </property>
  </action>
  <action name="actionIndex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Index Search</string>
   </property>
  </action>
  <action name="actionKeywords">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Keyword Search</string>
   </property>
  </action>
  <action name="actionPaths">
   <property name="text">
    <string>Man &amp;Paths</string>
   </property>
  </action>
  <action name="action_Mode">
   <property name="text">
    <string>Mode</string>
   </property>
   <property name="toolTip">
    <string>Change Search Mode</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>&amp;Reload</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
